modules/claims_api/lib/claims_api/poa_pdf_constructor/base.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require_dependency 'claims_api/stamp_signature_error'
```

Comment:
This has nothing to do with this ticket, however, it fixes failing tests, which would have blocked the merge.